.github/semantic.yml

```diff
@@ -32,3 +32,4 @@ scopes:
   - revert
   - tests
   - webpack
+  - prettier
```

Comment:
this now redundant that its in this PR.

https://github.com/f1v/full-git-commit-history-app/blob/53da35c00f49da0c7681933a6c7d3fe9d698832a/.github/semantic.yml#L30-L35